account-validator-api/src/java/org/sakaiproject/accountvalidator/logic/ValidationLogic.java

```diff
@@ -72,7 +72,17 @@
 	 * @return
 	 */
 	public ValidationAccount createValidationAccount(String UserId, boolean newAccount);
-	
+
+
+	/**
+	 *  Create a new Validation account
+	 * @param userRef existing userId
+	 * @param newUserId is the new id which the user wants to have as userId
+	 * @param accountStatus
+	 * @return
+	 */
+	public ValidationAccount createValidationAccount(String userRef,Integer accountStatus,String newUserId);
```

Comment:
because it is being called by other method CreateValidationAccount(userId,accountStatus)